### PR TITLE
demo function for mailchimp subscribe

### PR DESCRIPTION
**Pull request by benceHornyak**
Resolves: appwrite/appwrite#1914

## Execution log (fraction of the screenshot)

image

### CODE REVIEW

File: nodejs/subscribe-to-mailchimp/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# 📧 Subscribe to mailchimp newsletter using Mailchimp API V3
+A sample nodejs Cloud Function for subscribing a mailchimp newsletter
```
Comment (christyjacob4) [inline nodejs/subscribe-to-mailchimp/README.md:2]:
```suggestion
A sample nodejs Cloud Function for subscribing to a Mailchimp newsletter
```

File: nodejs/subscribe-to-mailchimp/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## 📝 Environment Variables
+Go to Settings tab of your Cloud Function. Add the following environment variables.
+
+* **APPWRITE_MAILCHIMP_API_KEY** - Your Mailchimp API key
+* **APPWRITE_MAILCHIMP_LIST_ID** - Your Mailchimp List ID
+
+Your data must include the following field:
```
Comment (christyjacob4) [inline nodejs/subscribe-to-mailchimp/README.md:10]: Mention specifically that the `APPWRITE_FUNCTION_DATA` variable must contain this data

File: nodejs/subscribe-to-mailchimp/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+```bash
+$ cd demos-for-functions/nodejs/subscribe-to-mailchimp
+$ tar -zcvf code.tar.gz .
+```
+
+* Navigate to the Overview Tab of your Cloud Function > Deploy Tag
+* Input the command that will run your function (in this case "node index.js") as your entrypoint command
+* Upload your tarfile 
+* Click 'Activate'
```
Comment (christyjacob4) [inline nodejs/subscribe-to-mailchimp/README.md:38]: Also add a ## Trigger section and mention how this function can be triggered like the other examples we already have in the repo.

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

**Review by Meldiron — APPROVED** [review]
Looks good to me

**Comment by Meldiron** [comment]
THANK YOU! All changes merged 🥳

Please reach out to `eldadfux` on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**